.github/workflows/release-sync-ami.yml

```diff
@@ -26,13 +26,26 @@ jobs:
       - name: Set AMI from private to public in all regions
         run: |
           #!/bin/bash
+ 
+          # The actions aws credential above would lock region
+          # This is to allow aws to switch region in order to 
+          # make AMI public on all available regions
+          aws configure set aws_access_key_id ${{ secrets.AWS_EC2_ACCESS_KEY_ID }}
+          aws configure set aws_secret_access_key ${{ secrets.AWS_EC2_SECRET_ACCESS_KEY }}
+          aws configure set region us-east-1
```

Comment:
we don't need set region here. can simply put region in the specific command line e.g. aws ec2 --region us-west-2

Reply:
This is a quick fix to release AMI.
Will follow up and set a better way later.